Makefile

```diff
@@ -7,6 +7,7 @@ SHAREDIR ?= ${PREFIX}/share
 PYTHON ?= $(shell command -v python3 python|head -n1)
 DESTDIR ?= /
 PATH := $(PATH):$(HOME)/.local/bin
+MYPIP := pipx
```

Comment:
Why not using `MYPIP ?= pipx` here?

It will be equivalent, but, in case anybody wants to override it externally, it will be possible to do so:

```bash
MYPIP=pip make unit-test
```

Reply:
Thanks! nice catch!